students/rgpag/lesson04/mailroom.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# initialize lists
+import datetime
+
+donors = ['Jeff Bezos', 'Mark Zuckerberg', 'Bill Gates', 'Paul Allen']
+i_amount = [3000, 20, 40607, 65]
+i_count = [4, 2, 3, 6]
+quit = 0
+
+# initialize amt_dict
+amt_dict = {}
+cnt_dict = {}
+avg_dict = {}
+i = 0
+for i in range(len(donors)):
+    amt_dict[donors[i]] = i_amount[i]
+    cnt_dict[donors[i]] = i_count[i]
+    avg_dict[donors[i]] = i_amount[i]/i_count[i]
+
+
+msg = """
+\nDear {},\n\nThank you for your recent donation, did you \
+realize you have now made {} lifetime donations? Wow, look at you all star! \
+We are most grateful for your total donation amount of ${}.\n\nI will \
+personally ensure these funds are put towards purchase of yummy \
+donuts! Also, any additional contributions you make in the next 24 hours \
+will be matched up to $1000. What a deal! Don't delay.\n\nHumbly yours,\n
+Dime for Donuts\n
+"""
+
+
+def new_donor(new_name):
+    donors.append(new_name)
+    don_amt = int(input('What was the donation amount?'))
+    amt_dict[new_name] = don_amt
+    cnt_dict[new_name] = 1
+    avg_dict[new_name] = don_amt/1
+    return new_name, 1, don_amt
+
+
+def update_don(ret_don):
+    don_amt = int(input('What was the donation amount?'))
+    if don_amt == 0:
+        return ret_don, cnt_dict[ret_don], amt_dict[ret_don]
+    cnt_dict[ret_don] += 1
+    amt_dict[ret_don] += don_amt
+    avg_dict[ret_don] = amt_dict[ret_don]/cnt_dict[ret_don]
+    return ret_don, cnt_dict[ret_don], amt_dict[ret_don]
+
+
+def getKey(item):
+    return item[1]
+
+
+def menu_sel_1():
+    name = input('Please provide full name ')
+    if name == 'quit':
+        quit = 1
```

Comment:
This is overriding the Python built-in `quit` function (see how GitHub highlights in blue like other built-ins, `input`, `print` etc.)